pkg/cnb/builder_builder.go

```diff
@@ -83,12 +80,27 @@ func (bb *BuilderBuilder) writeableImage() (v1.Image, error) {
 		layers = append(layers, layer.v1Layer)
 	}
 
-	orderLayer, err := bb.tomlLayer()
+	stackLayer, err := bb.stackLayer()
+	if err != nil {
+		return nil, err
+	}
+
+	orderLayer, err := bb.orderLayer()
 	if err != nil {
 		return nil, err
 	}
 
-	image, err := mutate.AppendLayers(bb.baseImage, append(layers, orderLayer)...)
+	lifecycleLayers, err := bb.lifecycleImage.Layers()
+	if err != nil {
+		return nil, err
+	}
+
+	image, err := mutate.AppendLayers(bb.baseImage, append(lifecycleLayers, stackLayer)...)
```

Comment:
Do we need pack's "default dirs" layer? 

https://github.com/buildpacks/pack/blob/e4315be24c103e3c5722b08561bf13a55876cbbc/internal/builder/builder.go#L481

Reply:
We probably do. Good call!